src/shared-content/administration/high-availability-shared-storage-overview.include.md

```diff
@@ -4,6 +4,11 @@ Octopus stores several files that are not suitable to store in the database. The
 - [Artifacts](/docs/projects/deployment-process/artifacts) collected during a deployment. Teams using Octopus sometimes use this feature to collect large log files and other files from machines during a deployment.
 - Task logs are text files that store all of the log output from deployments and other tasks.
 - Imported zip files used by the [Export/Import Projects feature](/docs/projects/export-import).
+- Archived audit logs by the [Archived audit logs feature](/docs/security/users-and-teams/auditing/index.md#archived-audit-events).
```

Comment:
I think with Astro (the new site "engine"), you don't need to include the index.md extension for links
```suggestion
- Archived audit logs by the [Archived audit logs feature](/docs/security/users-and-teams/auditing/#archived-audit-events).
```